test/Quake/control_and_adjoint.qke

```diff
@@ -0,0 +1,88 @@
+// ========================================================================== //
```

Comment:
We could merge this with adjoint-1.qke as they are testing the same passes.

Reply:
There's another problem in issue#1871 which I was planning to expand in this file as combinations of `control` and `adjoint`, hence prefer to keep it separate.